.travis.yml

```diff
@@ -4,7 +4,6 @@ elixir:
   - 1.5.3
   - 1.6.4
 otp_release:
-  - 18.3
```

Comment:
scaling native to nano seconds threw an error on OTP 18 so I'm inclined to believe it doesn't support it. As newer elixir versions also drop support I thought it was ok for us to do so as well.